handlers/guest_links_test.go

```diff
@@ -346,3 +345,193 @@ func TestDeleteInvalidGuestLink(t *testing.T) {
 		t.Fatalf("DELETE returned wrong status code: got %v want %v", status, http.StatusBadRequest)
 	}
 }
+
+func TestGuestLinkToggleInValidRequest(t *testing.T) {
+	type testCase struct {
+		description      string
+		operationUrls    []string
+		guestLinkInStore picoshare.GuestLink
+		expected         picoshare.GuestLink
+	}
+
+	for _, tt := range []testCase{
+		{
+			description: "guest_link operation: disable",
+			operationUrls: []string{
+				"/api/guest-links/abcdefgh23456789/disable",
+			},
+			guestLinkInStore: picoshare.GuestLink{
+				ID:             picoshare.GuestLinkID("abcdefgh23456789"),
+				Created:        mustParseTime("2024-01-01T00:00:00Z"),
+				UrlExpires:     mustParseExpirationTime("2030-01-02T03:04:25Z"),
+				FileLifetime:   picoshare.FileLifetimeInfinite,
+				MaxFileBytes:   picoshare.GuestUploadUnlimitedFileSize,
+				MaxFileUploads: picoshare.GuestUploadUnlimitedFileUploads,
+			},
+			expected: picoshare.GuestLink{
+				ID:             picoshare.GuestLinkID("abcdefgh23456789"),
+				Created:        mustParseTime("2024-01-01T00:00:00Z"),
+				UrlExpires:     mustParseExpirationTime("2030-01-02T03:04:25Z"),
+				FileLifetime:   picoshare.FileLifetimeInfinite,
+				MaxFileBytes:   picoshare.GuestUploadUnlimitedFileSize,
+				MaxFileUploads: picoshare.GuestUploadUnlimitedFileUploads,
+				IsDisabled:     true,
+			},
+		},
+		{
+			description: "guest_link operation: enable",
```

Comment:
I'd describe this one as "enabling guest link succeeds but has no effect when guest link is not disabled"